sumologic/resource_sumologic_partition.go

```diff
@@ -29,7 +29,7 @@ func resourceSumologicPartition() *schema.Resource {
 			"analytics_tier": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"enhanced", "basic", "cold"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"continuous", "frequent", "infrequent"}, false),
 				Default:      "enhanced",
```

Comment:
Default shouldn't be enhanced then right? 
And I remember seeing somewhere that the API is backwards compatible so Terraform should accept old and new probably.